tensorflow/compiler/xla/service/gpu/cudnn_fused_conv_rewriter_test.cc

```diff
@@ -40,24 +40,35 @@ class CudnnFusedConvRewriterTest : public HloTestBase {
   }
 
   void TestMatchWithAllTypes(absl::string_view hlo_string) {
-    for (absl::string_view type : {"f16", "f32", "f64"}) {
-      const string hlo_with_new_type =
-          absl::StrReplaceAll(hlo_string, {{"TYPE", type}});
-      string optimized_hlo_string = GetOptimizedHlo(hlo_with_new_type);
+    string alpha_conv_scalar, alpha_side_input_scalar;
+    for (absl::string_view type : {"f16", "f32", "f64", "s8"}) {
+      if(type == "s8") {
+        alpha_conv_scalar = "2";
+        alpha_side_input_scalar = "-3";
+      }
+      else {
+        alpha_conv_scalar = "0.999994934";
+        alpha_side_input_scalar = "0.899994934";
+      }
+      string hlo_resolved_string =
+          absl::StrReplaceAll(hlo_string, {{"TYPE", type}, {"ALPHA_CONV_SCALAR", alpha_conv_scalar}, {"ALPHA_SIDE_INPUT_SCALAR", alpha_side_input_scalar}});
+      string optimized_hlo_string = GetOptimizedHlo(hlo_resolved_string);
       EXPECT_THAT(optimized_hlo_string,
                   Not(HasSubstr(kCudnnConvForwardCallTarget)));
       EXPECT_THAT(optimized_hlo_string,
                   HasSubstr(kCudnnConvBiasActivationForwardCallTarget));
-      EXPECT_TRUE(RunAndCompare(hlo_with_new_type, ErrorSpec{0.01}))
-          << optimized_hlo_string;
+      if (type != "s8") {
```

Comment:
Skip numerical verification for s8 until the int8 HLO evaluator visitor has proper semantics.

Reply:
Ideally we should fix the evaluator before checking in this patch. Do you plan to fix it?

Reply:
Yes, I am working on it.

Reply:
int8-to-int8 is disallowed and int8-to-float is implemented in the follow-up commit